synex-frontend/src/codebase/indexer.ts

```
@@ -1,14 +1,397 @@
+  }
+
+  // Extract import statements
+  private extractImports(content: string, extension: string): string[] {
+    const imports: string[] = [];
+
+    switch (extension) {
+      case '.js':
+      case '.jsx':
+      case '.ts':
+      case '.tsx':
+        const jsImportRegex = /import\s+(?:[^\n]+\s+from\s+)?['"]([^'"]+)['"]/g;
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)